lib/board.rb

```diff
@@ -80,6 +79,9 @@ def placement_consecutive?
       p "Ooops"
     end
   end
+# DIAGONAL METHOD HERE
```

Comment:
Do we need a diagonal method? I'm confused about this. I thought our valid suite covered diagonal coordinates. Maybe I'm misunderstanding this comment line and we might need to chat. Let me know :)

Reply:
Lol!  Hey, I saw you type this to the screen a while ago and we must've forgotten it was there.